addons/nuxeo-ai-aws-core/src/main/java/org/nuxeo/ai/endpoint/Rekognition.java

```diff
@@ -60,8 +60,8 @@
 /**
  * Endpoints responsible for listening notifications from external services such as AWS SNS
  */
-@Path("/ai/rekognition")
-@WebObject(type = "ai")
+@Path("/aiaddons/rekognition")
```

Comment:
I think we should try to make the API common throughout the add-on. We don't need to cross import the projects. As you remember we can spawn a web object by its type, like we do in the [cloud](https://github.com/nuxeo/nuxeo-ai-cloud/blob/master/ai-cloud-jaxrs/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/AIWebObject.java#L430). 
Then we'd have endpoints `ai/config/...` and `ai/rekognition`, and so on

Reply:
I tried and like i told you we cant put a dependency on nuxeo-ai-config

Reply:
but we don't need to. WebObject will be resolved by simple string name

Reply:
You need to return the web adapter of aws in the AIRoot.